src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -4,6 +4,7 @@
 using Azure.Core;
 using Azure.Data.AppConfiguration;
 using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions;
 using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
```

Comment:
What is this using added for?

Reply:
It's for using `NormalizeNull()` to compare labels [here](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/234/files/49994711cbad21f0ddfa36c5e44256655f4dfec4#diff-9fe8bf4d21db7a9a52895366df1e58edce9f4d7dca80514f7b457311f3218cecR186).

Reply:
@avanigupta I see, thanks!